### PR TITLE
Add Account / password changing

### PR DESCRIPTION
**Pull request by vadi2**
image

### CODE REVIEW

File: controllers/account.lua [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  end),
+  GET = capture_errors(function(self)
+    assert_error(self.session.name, self.i18n("err_not_logged_in"))
+    return { render = "account" }
+  end),
+  POST = capture_errors(function(self)
+    for k,v in pairs(self.params) do ngx.log(ngx.ERR,k) end
+    ngx.log(ngx.ERR, "done printing params")
+    validate.assert_valid(self.params, {
+      { "new_password", exists = true, min_length = 8, "Please enter the new password." },
```
Comment (vadi2) [inline controllers/account.lua]: This is being problematic - no `new_password` is getting passed in :thinking:

Comment (vadi2) [reply controllers/account.lua]: The issue was that my elements didn't have the `name` on them.

File: controllers/account.lua [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  POST = capture_errors(function(self)
+    validate.assert_valid(self.params, {
+      { "new_password", exists = true, min_length = 8, i18n("missing_new_password") },
+      { "confirm_new_password", equals = self.params.new_password,
+            i18n("confirmation_password_must_match") },
+    })
+
+    self.params.password = self.params.old_password
+    self.params.name = self.session.name
+    assert_error(Users:verify_user(self.params), i18n("old_password_mismatch"))
```
Comment (vadi2) [inline controllers/account.lua:23]: I'm not a big fan of hacking `self.params` to please `Users:verify_user` - the API would be better if the latter just took in the username and password as it expects. Right now, we have to leak `self.params` into it. What do you think @demonnic?

File: views/stylesheet.etlua [hunk 3]
```diff
@@ -77,14 +77,18 @@
...
 
-  * html main{ 
-    height: 100%; 
-    width: 100%; 
+  * html main{
+    height: 100%;
+    width: 100%;
+  }
+
+  .form-control {
```
Comment (vadi2) [inline views/stylesheet.etlua:91]: This is the only change here, the rest is trimming whitespace at the end of lines.

File: controllers/account.lua (hunk 2 above)
Comment (demonnic) [reply controllers/account.lua:23]: That's fine. When I wrote it self.params was just a table containing the username and password. If you want to refactor it to extract the uname and pass from the table and send those specifically go for it.

Comment (demonnic) [reply controllers/account.lua:23]: would you rather do that in this PR or another?

Comment (vadi2) [reply controllers/account.lua:23]: Another

**Review by demonnic — APPROVED** [review]

**Pull request merged**